Networking/NetworkingTests/Remote/SitePluginsRemoteTests.swift

```diff
@@ -64,4 +64,142 @@ class SitePluginsRemoteTests: XCTestCase {
             XCTAssertNotNil(error)
         }
     }
+
+    // MARK: - Install plugin tests
+
+    /// Verifies that installPlugin properly parses the sample response.
+    ///
+    func test_installPlugin_properly_returns_plugins() {
+        let remote = SitePluginsRemote(network: network)
+
+        network.simulateResponse(requestUrlSuffix: "plugins", filename: "plugin")
+
+        // When
+        let result: Result<SitePlugin, Error> = waitFor { promise in
+            remote.installPlugin(for: self.sampleSiteID, slug: "jetpack") { result in
+                promise(result)
+            }
+        }
+
+        // Then
+        switch result {
+        case .success(let plugin):
+            XCTAssertEqual(plugin.plugin, "jetpack/jetpack")
+        case .failure(let error):
+            XCTAssertNil(error)
+        }
+    }
+
+    /// Verifies that installPlugin properly relays Networking Layer errors.
+    ///
+    func test_installPlugin_properly_relays_netwoking_errors() {
+        let remote = SitePluginsRemote(network: network)
+
+        // When
+        let result: Result<SitePlugin, Error> = waitFor { promise in
+            remote.installPlugin(for: self.sampleSiteID, slug: "jetpack") { result in
+                promise(result)
+            }
+        }
+
+        // Then
+        switch result {
+        case .success(let plugin):
+            XCTAssertNil(plugin)
+        case .failure(let error):
+            XCTAssertNotNil(error)
+        }
+    }
+
+    // MARK: - Activate plugin tests
+
+    /// Verifies that activatePlugin properly parses the sample response.
+    ///
+    func test_activatePlugin_properly_returns_plugins() {
+        let remote = SitePluginsRemote(network: network)
+
+        network.simulateResponse(requestUrlSuffix: "plugins/jetpack/jetpack", filename: "plugin")
+
+        // When
+        let result: Result<SitePlugin, Error> = waitFor { promise in
+            remote.activatePlugin(for: self.sampleSiteID, pluginName: "jetpack/jetpack") { result in
+                promise(result)
+            }
+        }
+
+        // Then
+        switch result {
+        case .success(let plugin):
+            XCTAssertEqual(plugin.plugin, "jetpack/jetpack")
+        case .failure(let error):
+            XCTAssertNil(error)
+        }
+    }
+
+    /// Verifies that activatePlugin properly relays Networking Layer errors.
+    ///
+    func test_activatePlugin_properly_relays_netwoking_errors() {
+        let remote = SitePluginsRemote(network: network)
+
+        // When
+        let result: Result<SitePlugin, Error> = waitFor { promise in
+            remote.activatePlugin(for: self.sampleSiteID, pluginName: "jetpack/jetpack") { result in
+                promise(result)
+            }
+        }
+
+        // Then
+        switch result {
+        case .success(let plugin):
+            XCTAssertNil(plugin)
+        case .failure(let error):
+            XCTAssertNotNil(error)
+        }
```

Comment:
nit: similar to above
```suggestion
        XCTAssertTrue(result.isFailure)
```